### DISCUSSION TITLE
Help with Czech and French translation

### DISCUSSION THREAD

**Discussion by anthidote** [alfio-event/alf.io#1262] [translations]
Hello, I would like to help with the Czech and French translation. I have applied to join the projects in POEditor

**Comment by cbellone** [comment]
Hi @anthidote ,

done! Thank you very much for helping us! :+1:

**Reply by anthidote** [reply]
@cbellone Hi, I've finished the Czech translation. Could you please include it in the next release? We'd like to test it out in our community project. Thanks!

There's a big chunk of the French translation that I can do, simple things, but definitely at least 100-200 items that will need to be done by a native speaker.

**Reply by cbellone** [reply]
Hi,
Thank you for translating to Czech!
Will integrate the new translation as soon as possible and report back here as soon as it is available.

> There's a big chunk of the French translation that I can do, simple things, but definitely at least 100-200 items that will need to be done by a native speaker.

I am not sure I get what you mean. Are you referring to the missing translations or the existing ones?
FYI French translation was contributed by some native speakers
